hail_search/test_search.py

```diff
@@ -365,34 +368,35 @@ async def test_both_sample_types_search(self):
             MULTI_PROJECT_BOTH_SAMPLE_TYPE_VARIANTS, gene_counts=GENE_COUNTS, sample_data=MULTI_PROJECT_SAMPLE_TYPES_SAMPLE_DATA,
         )
 
-        # Variant1 in family_2 is de novo in exome but maternally inherited in genome.
-        # Genome passes quality and inheritance, show genotypes for both sample types.
-        variant1_interval = ['1', 10438, 10440]
+        # Variant 1 is de novo in exome but maternally inherited in genome.
+        # Variant 2 is inherited in exome and de novo in genome.
```

Comment:
I would clarify in the comment that inherited means "inherited and homozygous" as usually maternally inherited means mom has one alt allele and proband inherited that and has 1 alt allele

Reply:
You updated the comment for variant 1 but not variant 2, de novo is usually a het call so its still confusing

Reply:
I think having the ability to have at least one variant passing for homozygous recessive search is valuable for testing and I like the coverage we get from this variant configuration.  And we do sometimes see de novo homozygotes, we just don't refer to it as plain "de novo" which is why I think a comment update is sufficient